packages/webapp/pages/account/profile.tsx

```diff
@@ -135,7 +135,7 @@ const AccountProfilePage = (): ReactElement => {
     </span>
   );
 
-  const RenderForm = () => (
```

Comment:
Producing an infinite loop. Either we probably wrap it with `useMemo` or the changes made here.

Reply:
This seems weird, we should be very careful on why this happens.

Reply:
I think the components within are getting re-mounted since it is a function and gets recreated at every state update, then some `useEffect` or some that do state change executes which loops it back.

This actually fixes the current issue about changing the timezone would reset everything.